docs/guides/hanadb/quickstart/quickstart.md

```
@@ -0,0 +1,269 @@
+kubernetes.io/basic-auth
+
+$ kubectl get secret -n demo hanadb-quickstart-auth -o go-template='{{range $k,$v := .data}}{{$k}}{{"\n"}}{{end}}'
+password
+password.json
+username
+```
+
+Read the password into a shell variable (avoid pasting real passwords into shared terminals):
+
+```bash
+$ HANA_PASSWORD="$(kubectl get secret hanadb-quickstart-auth -n demo -o jsonpath='{.data.password}' | base64 -d)"
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)